reactor-netty-core/src/main/java/reactor/netty/transport/ProxyProvider.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017-2024 VMware, Inc. or its affiliates, All Rights Reserved.
+ * Copyright (c) 2017-2025 VMware, Inc. or its affiliates, All Rights Reserved.
```

Comment:
No changes in the file 

```suggestion
 * Copyright (c) 2017-2024 VMware, Inc. or its affiliates, All Rights Reserved.
```